pdns/recpacketcache.cc

```diff
@@ -190,20 +198,17 @@ uint64_t RecursorPacketCache::size()
 
 uint64_t RecursorPacketCache::bytes()
```

Comment:
Could be marked `const`.

Reply:
pushed; also added const to a few others that did not have it before.